plugins/command-manager/src/main/java/com/wazuh/commandmanager/jobscheduler/SearchThread.java

```diff
@@ -223,12 +176,12 @@ public SearchResponse pitQuery(PointInTimeBuilder pointInTimeBuilder, Object[] s
             this.searchSourceBuilder.searchAfter(searchAfter);
         }
         searchRequest.source(this.searchSourceBuilder);
+
         return this.client.search(searchRequest).actionGet(timeout);
     }
 
     @Override
     public void run() {
-        log.debug("Running scheduled job");
```

Comment:
I think it would be great to keep this debug log to be aware of how many times this method is invoked.